test/libdnf/rpm/repo/test_repo.cpp

```diff
@@ -60,13 +60,14 @@ void RepoTest::test_repo_basics() {
     libdnf::rpm::SolvSack sack(base);
 
     // Creates system repository and loads it into rpm::SolvSack.
-    sack.create_system_repo(false);
+    // TODO(dmach): avoid loading system repo from host
```

Comment:
I've updated the TODO comment. I'd leave the next line commented, because I don't know how to fix it properly. I propose dealing with it later when we do a review of TODOs.

Reply:
Sure. But I actually don't know what's the outstanding issue - you've stopped loading the system repo by commenting out the line. So what is the desired state, to have a mocked system repo for the tests? I'd like to have that described in the TODO, my point is right now I don't know what the TODO is about...